zerver/tests/test_user_status.py

```diff
@@ -312,3 +397,24 @@ def test_endpoints(self) -> None:
             user_status_info(hamlet),
             dict(status_text="at the beach", away=True),
         )
+
+        result = self.client_get(f"/json/users/{hamlet.id}/status", {})
+        self.assert_json_success(result)
+        result_dict = result.json()
+        self.assertEqual(
+            result_dict["status"],
+            dict(status_text="at the beach", away=True),
+        )
+
+        # Invalid user ID should fail
+        result = self.client_get("/json/users/12345/status", {})
+        self.assert_json_error(result, "No such user")
+
+        # Test status if it has not been set
```

Comment:
Isn't this tested above when the status is cleared?

Reply:
Yes, it is. I have removed it from here.

Reply:
I think so, but the coverage tests fail if I remove that test.

Reply:
I think so, but the coverage tests fail if I remove that check.

Reply:
Cool, thanks for checking. Leaving the test for coverage makes sense. And it's clearly labeled as to what it's doing.